src/service/navigation.js

```diff
@@ -566,6 +586,10 @@ export class Navigation {
       ) {
         this.removeViewerQueryBeforeNavigation_(win, fromLocation, target);
       }
+
+      if (this.navigateToUrlWithViewer(to, 'external nav')) {
```

Comment:
Is "external nav" desired? How about something like "intercept_click"?

Reply:
Done.